lib/namae/parser.y

```diff
@@ -20,7 +20,7 @@ rule
        | sort_order
 
   honorific : APPELLATION { result = Name.new(:appellation => val[0]) }
-            | TITLE       { result = Name.new(:title => val[0]) }
+            | titles      { result = Name.new(:title => val[0]) }
```

Comment:
I think this is fine, but just to make sure: is this required for the particle parsing or just in general to allow multiple titles?

Reply:
Just to allow multiple titles.